synse/http_integration_test.go

```diff
@@ -0,0 +1,473 @@
+package synse
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vapor-ware/synse-client-go/synse/scheme"
+)
+
+func TestIntegration_Status(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	client, err := NewHTTPClientV3(&Options{
+		Address: "localhost:5000",
+	})
+	assert.NotNil(t, client)
+	assert.NoError(t, err)
+
+	status, err := client.Status()
+	assert.NoError(t, err)
+	assert.Equal(t, "ok", status.Status)
+	assert.NotEmpty(t, status.Timestamp)
+}
+
+func TestIntegration_Version(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	client, err := NewHTTPClientV3(&Options{
+		Address: "localhost:5000",
+	})
+	assert.NotNil(t, client)
+	assert.NoError(t, err)
+
+	version, err := client.Version()
+	assert.NoError(t, err)
+	assert.Equal(t, "3.0.0-alpha.3", version.Version)
```

Comment:
this is a case where just asserting that it is not empty is fine, since this value will change based on whatever the `:latest` tag is. the alternative is to pin the tag to a known version in the compose file, but I think that could lead to problems down the line